src/react_native/utils.cljs

```diff
@@ -0,0 +1,20 @@
+(ns react-native.utils)
+
+(defonce throttle (atom {}))
+
+(defn throttled-on-press
```

Comment:
This function is not part of the public API we want to expose, so we can also make it private.